testing/manual/index.js

```diff
@@ -32,13 +31,25 @@ function processGSUB(GSUB) {
       features.forEach((feature) => {
         const lookupIDs = feature.lookupListIndices;
 
+        // TODO: FIXME: this seems to be finding GPOS lookups, which shouldn't show up for GSUB?
```

Comment:
I noticed this too, for example it always reports GPOS `kern` as a GSUB feature in Wakamai Fondue (still does with this version)